docs/configuration/validation/brk.rst

```diff
@@ -0,0 +1,43 @@
+.. _configuration_validation_brk:
+
+=========================
+BRK - Zakelijk gerechtigd
+=========================
+
+The special *AddressNL* component has a specific validator available, that can
+check if the currently logged in user is the business owner of the filled-in property.
+
+Configuration
+=============
+
+1. Obtain a BRK API key.
+2. In Open Forms, navigate to: **Miscellaneous** > **KvK configuration**.
+3. Click **Add** next to the **KvK API Basisprofiel** field and fill in the following
+   details:
+
+   * **Label**: *Fill in a human readable label*, for example: ``My BRK API``
+   * **Type**: ORC (Overige).
+   * **API root URL**: *The API URL provided by KvK but typically:* ``https://api.brk.kadaster.nl/esd-eto-apikey/bevragen/v2/``.
+   * **Authorization type**: API key.
+   * **Header key**: ``X-Api-Key``.
+   * **Header value**: *The KvK API key obtained in step 1*.
+   * **OAS file**: ``https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-BRK-bevragen/master/specificatie/genereervariant/openapi.yaml``.
```

Comment:
this is only required for legacy reasons right, we don't actually make use of this file anymore?

Reply:
I thought at first a required field, glad to remove it as it did not play well with the RTD theme